src/Fixer/Operator/OperatorLinebreakFixer.php

```diff
@@ -0,0 +1,262 @@
+            if (null === $indices) {
+                continue;
+            }
+
+            $nextIndex = $tokens->getNextMeaningfulToken(max($indices));
+            for ($i = $nextIndex - 1; $i > $index; --$i) {
+                if ($tokens[$i]->isWhitespace() && 1 === Preg::match('/\R/u', $tokens[$i]->getContent())) {
+                    $operators = $this->getReplacementsAndClear($tokens, $indices, -1);
+                    $tokens->insertAt($nextIndex, array_merge($operators, [new Token([T_WHITESPACE, ' '])]));
+
+                    break;
+                }
+            }
+            $index = $nextIndex;
+        }
+    }
+
+    private function fixMoveToTheEnd(Tokens $tokens)
+    {
+        for ($index = $tokens->count() - 1; $index > 0; --$index) {
+            $indices = $this->getOperatorIndices($tokens, $index);
+            if (null === $indices) {
+                continue;
+            }
+
+            $prevIndex = $tokens->getPrevMeaningfulToken(min($indices));
+            for ($i = $prevIndex + 1; $i < $index; ++$i) {
+                if ($tokens[$i]->isWhitespace() && 1 === Preg::match('/\R/u', $tokens[$i]->getContent())) {
+                    $operators = $this->getReplacementsAndClear($tokens, $indices, 1);
+                    $tokens->insertAt($prevIndex + 1, array_merge([new Token([T_WHITESPACE, ' '])], $operators));
+
+                    break;
+                }
+            }
+            $index = $prevIndex;
+        }
+    }
+
+    /**
+     * @param Tokens $tokens
+     * @param int    $index
+     *
+     * @return null|int[]
+     */
+    private function getOperatorIndices(Tokens $tokens, $index)
+    {
+        if (null === $this->operators) {
+            $this->operators = self::BOOLEAN_OPERATORS;
+            if (!$this->onlyBooleans) {
+                $this->operators += self::NON_BOOLEAN_OPERATORS;
+            }
+        }
+
+        if (!isset($this->operators[\strtolower($tokens[$index]->getContent())])) {
+            return null;
+        }
+
+        if (isset($this->operators['?']) && '?' === $tokens[$index]->getContent()) {
+            $nextIndex = $tokens->getNextMeaningfulToken($index);
+            if (':' === $tokens[$nextIndex]->getContent()) {
+                return [$index, $nextIndex];
+            }
+        }
+
+        if (isset($this->operators[':']) && ':' === $tokens[$index]->getContent()) {
+            $prevIndex = $tokens->getPrevMeaningfulToken($index);
+            if ('?' === $tokens[$prevIndex]->getContent()) {
+                return [$prevIndex, $index];
+            }
+        }
+
+        return [$index];
+    }
+
+    /**
+     * @param Tokens $tokens
+     * @param int[]  $indices
+     * @param int    $direction
+     *
+     * @return Token[]
+     */
+    private function getReplacementsAndClear(Tokens $tokens, array $indices, $direction)
+    {
+        return array_map(
+            function ($index) use ($tokens, $direction) {
```

Comment:
this can be `static function` I think? no need to leak the scope in here

Reply:
I thought we have `static_lambda` set up for the repo.